src/hotspot/os/aix/os_aix.cpp

```diff
@@ -619,66 +602,42 @@ void os::init_system_properties_values() {
 bool os::Aix::get_meminfo(meminfo_t* pmi) {
 
   assert(pmi, "get_meminfo: invalid parameter");
-
   memset(pmi, 0, sizeof(meminfo_t));
 
-  if (os::Aix::on_pase()) {
-    // On PASE, use the libo4 porting library.
-
-    unsigned long long virt_total = 0;
-    unsigned long long real_total = 0;
-    unsigned long long real_free = 0;
-    unsigned long long pgsp_total = 0;
-    unsigned long long pgsp_free = 0;
-    if (libo4::get_memory_info(&virt_total, &real_total, &real_free, &pgsp_total, &pgsp_free)) {
-      pmi->virt_total = virt_total;
-      pmi->real_total = real_total;
-      pmi->real_free = real_free;
-      pmi->pgsp_total = pgsp_total;
-      pmi->pgsp_free = pgsp_free;
-      return true;
-    }
+  // On AIX, I use the (dynamically loaded) perfstat library to retrieve memory statistics
```

Comment:
maybe remove the "I" pronoun in the comment here? Also the links could be placed on one line.